pkg/sentry/arch/arch_state_x86.go

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// +build amd64
```

Comment:
amd64 i386

Reply:
(Though we won't support i386, if the file is named _x86.go then it should be both)

Reply:
ok.